config/crd/bases/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -772,6 +772,12 @@ spec:
                       SYS_RESOURCE.'
                     type: boolean
                   resources:
+                    default:
+                      limits:
+                        memory: 100Mi
+                      requests:
+                        cpu: 100m
+                        memory: 50Mi
```

Comment:
I think these defaults for memory would be too low for eBPF. It might be okay for RSS but I have seen container_working_set_bytes for ebpf pods in the order of around 250-300 Mb.  It gets OOMKilled when container_working_set_bytes reaches limits.

Reply:
I guess we can raise that a bit but as said in the scrum today I think we should keep these values as low as possible upstream since we don't know the cluster size.
We need to add documentation to explain expected values per components per expected amount of flows so the user will be able to override this wisely.